package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "stijl",
-  "version": "2.9.2",
-  "description": "Generieke styling voor data.amsterdam.nl",
+  "version": "3.0.0",
+  "description": "Generic styling voor data.amsterdam.nl",
```

Comment:
"Generieke" -> "Generic", dan ook "voor"  -> "for", right?

Reply:
dank!